ci/hardware_tests/record_video.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash 
+if [ $# -ne 2 ]
+  then
+    echo "Usage: $0 commit_id [start|stop]"
+fi
+
+OUTPUTFILE=video_$1_$(date +%s).mp4
+FONTFILE=$(ls /nix/store/*dejavu-fonts*/share/fonts/truetype/DejaVuSans-Bold.ttf|head -1)
+
+if [ "$2" == "start" ]; then
+  echo "[software/video] Starting record to $OUTPUTFILE"
+  ffmpeg -loglevel panic -f oss -f video4linux2 -i /dev/video0 \
+    -vf "drawtext=fontfile=$FONTFILE: \
```

Comment:
It should be enough to just use `"drawtext=font=Dejavu Sans: \` and we can avoid the NixOS specific `FONTFILE` assignment on line 8 above.

Reply:
Done. Btw maybe we can omit it completely, the default font seems fine too.